pkg/alertmanager/statefulset_test.go

```diff
@@ -33,9 +33,11 @@ import (
 var (
 	defaultTestConfig = Config{
 		ReloaderConfig: operator.ReloaderConfig{
-			Image:  "quay.io/prometheus-operator/prometheus-config-reloader:latest",
-			CPU:    "100m",
-			Memory: "25Mi",
+			Image:         "quay.io/prometheus-operator/prometheus-config-reloader:latest",
+			CPURequest:    "100m",
+			CPULimit:      "100m",
+			MemoryRequest: "50Mi",
+			MemoryLimit:   "50Mi",
 		},
```

Comment:
All tests that specify resources utilize "50Mi" for memory so I've bumped the defaultTestConfig to match.